lib/lz4.c

```diff
@@ -773,34 +846,66 @@ LZ4_FORCE_INLINE int LZ4_compress_generic(
         /* Fill table */
         LZ4_putPosition(ip-2, cctx->hashTable, tableType, base);
 
+#if 1
```

Comment:
Is there a reason to leave the `#if 1` around?

Reply:
I was making some speed comparison tests.
You're right, it shall be removed before merge.

Reply:
I've tried removing it as well, but it was a pretty large loss for me, though I don't remember the exact numbers, maybe 5%.

Reply:
yes, I had about the same experience